### PR TITLE
F ft 87 listing

### PR DESCRIPTION
**Pull request by adnaniroshanak**
Changes from listing and new validator

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -314,6 +314,92 @@ paths:
...
+    get:
+      tags:
+        - "Listing"
+      summary: "Get a facility with the given ID"
+      description: ""
+      security:
+        - firebase: []
+      operationId: "getFacilityListing"
+      x-google-backend:
+        address: "%%CLOUDFUNCTIONS_ENDPOINT_HOST%%/crud_facility_listings_http_fs_v1"
```
Comment (thees) [inline api.swagger.yaml:327]: missing?
path_translation: "APPEND_PATH_TO_ADDRESS"

File: api.swagger.yaml [hunk 2]
```diff
@@ -314,6 +314,92 @@ paths:
...
+    put:
+      tags:
+        - "Listing"
+      summary: "Update listings of a facility with the given ID"
+      description: ""
+      security:
+        - firebase: []
+      operationId: "putFacilityListing"
+      x-google-backend:
+        address: "%%CLOUDFUNCTIONS_ENDPOINT_HOST%%/crud_facility_listings_http_fs_v1"
```
Comment (thees) [inline api.swagger.yaml:367]: missing?
path_translation: "APPEND_PATH_TO_ADDRESS"

Comment (florian-bartels) [reply api.swagger.yaml:369]: i added them

File: api.swagger.yaml (hunk 1 above)
Comment (florian-bartels) [reply api.swagger.yaml:328]: added them

File: api-ctl.sh [hunk 3]
```diff
@@ -44,6 +44,12 @@ EOF
   $(wget -nv -O "stubs.zip" "${LINK}")
   unzip -o stubs.zip
   rm stubs.zip
+  cd  typescript-fetch-client
+  grep -n -A 64 "import \* as url" api.ts | sed -n 's/^\([0-9]\{1,\}\).*/\1d/p' | sed -f - api.ts  > api.tmp
+  grep -n -A 10000  -B 4 "export const " api.tmp  | sed -n 's/^\([0-9]\{1,\}\).*/\1d/p'  | sed -f - api.tmp > api2.tmp
+  tail -n +2 api2.tmp > api.ts
+  rm api2.tmp
+  rm api.tmp
```
Comment (evilass) [inline api-ctl.sh:52]: Some comments would be great here, because the commands are not self explanatory.

**Pull request merged**